Buch-Workshop.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Neue Poster für die senseBox – Coming Soon!"
+date: 2018-02-12
+author: Gina
+abstract: "Im Buch-Workshop wurden neue Poster zur einfachen Erklärung der senseBox für SchülerInnen, Studierende und LehrerInnen entwickelt."
+thumbnail: /images/blog_images/Buch-Workshop.png
+image1:/images/blog_images/IMG_0257
+lang: de
+---
+Zwei Tage lang rauchten die Köpfe
+============
+In einem Workshop der senseBox, unterstützt von der Initiative Make Light des Bundesministeriums für Bildung und Forschung, im Institut für Geoinformatik der Uni Münster überlegten rund 20 Teilnehmende, wie Lehr- und Lernmaterialien rund um die senseBox weiterentwickelt werden könnten. Damit kein heilloses Chaos entstand, sorgte Volker Bombien vom Bombini-Verlag als Moderator für einen strukturierten Ablauf. Außerdem erhielt das senseBox-Team Unterstützung von den Jungen Tüftlern, einem gemeinnützigen Unternehmen, das Kindern Digitalisierung, Technologien und den Umgang mit Daten näher bringen möchte.
+
+{% include image.html image=page.image %}
+
+Klärung der W-Fragen: Was und wen wollen wir wie erreichen?
+============
+Bei der Frage, was mit einer Publikation über die senseBox erreicht werden soll, ergab sich ein einheitliches Bild unter den Teilnehmenden: Der Nutzen der SenseBox insbesondere der Messwerte soll übersichtlich erklärt werden, um Verständnis für die technischen Möglichkeiten zu schaffen.Um auch "out of the box" zu denken, wurden kreative Ideen und Vorschläge für eine neue Publikation aus verschiedenen Blickwinkeln betrachtet. Als Anregung wurde in Impulsreferaten vorgestellt, wo die senseBox in Zukunft stehen könnte und welche Anfordernisse es an die Bildungsmaterialien für das senseBox-Projekt in den kommenden 3 Jahre geben wird. Ein großes Potenzial bietet der Einsatz der senseBox Institutionen wie Schulen, Hochschulen und Universiäten im Sinne einer fächerübergreifenden Anwendung. Daher soll ein leicht zugängliches Konzept für SchülerInnen, Studierende und Lehrende entwickelt werden, um zu zeigen, wie sie die Daten interpretieren können. Aber wie?
```

Comment:
senseBox mit kleinem s,

weiter hinten: Es fehlt ein Leerzeichen vor "Um auch 'out of the box' ...."